### ISSUE TITLE
default qmelt priority is too low resulting in slow export 

### ISSUE THREAD

**Issue by vrobolab** [mltframework/shotcut#830]
When exporting a video the default qmelt priority ("nice") sets to 10, which is way too low. If I manually renice the qmelt process to 0 the exporting becomes two or even three times faster, and it does not really interfere with other running applications.
Please consider making user-settable qmelt priority, or change it to higher value e.g. 1-5

(OS: openSUSE Linux, latest Shotcut)

**Comment by ddennedy** [comment]
nice is not supposed to make it run slower on a non-busy system, and certainly without nice it may interfere with other applications depending on number of cores and activity. Perhaps it is preventing CPU frequency from scaling up. I will take a look at it and strongly consider to add a priority option.

**Comment by ddennedy** [comment]
I tested this on my Ubuntu 19.10 system, and I did not see any worthwhile difference between `nice` and normal. My export jobs lasted roughly 30 minutes each with parallel processing enabled. I tested both with and without hardware encoder. I verified that CPU frequency scaling is enabled on my system, and it increases as desired even when using a combination of: nice 10, parallel processing, and hardware encoder. I am still not opposed to adding a priority option, but for now I might simply change the nice level to something like 3.
